### ISSUE TITLE
You can easily and safely mass murder an Outpost's crew with flooding

### ISSUE THREAD

**Issue by dylanstrategie** [Regalis11/Barotrauma#3833]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
The AI seems to be totally incapable of reacting to the Outpost being flooded if you put even minimal effort to cut in a place the AI can't immediately see and call out like a supply closet, or in some buggy spots like above catwalks, allowing you to flood the entire station and kill anyone without a mask.

**Steps To Reproduce**
- Spawn in Campaign
- Get a Plasma Cutter from your ship or steal one discreetly
- Find a place that's not being looked at by anyone and plasma cutter the wall until it floods properly
- Plasma Cutter all the doors in the way from the central well (even if seen, no-one will react unless you hurt people with the Plasma Cutter)
- Watch the entire Outpost flail uselessly and even pass right in front of the breach without fixing it because "need a diving suit", leading to half the Outpost dead within two minutes
- Leave with zero relations impact despite having destroyed the entire Outpost and killed half the crew

**Version**
V0.10.5.0 Unstable Build. Windows 10

**Additional information**
Engineers and Mechanics normally are perfectly competent at fixing any breaches you intentionally put into the Outpost if they see the hole being made or are called to it, but if the room is flooded without anyone seeing it they seem to become totally incapable of intervening.

I assume it's a mix of Outposts not having diving suits, the AI refusing to sacrifice themselves to fix even a one wall breach, and the player being allowed to casually plasma cutter all around the station, destroying doors and walls without any repercussions.

Even if AIs see the breach after it's made, they seem to ignore or fail to repair it. This might be due to the excessive flooding putting them on a mad search for a diving suit, which they can't find. Most of them will just flee to an unflooded room, often spreading the flood in the process.

I also noted that the AIs with masks never die due to free infinite mask + O2, and apparently the pressure never gets high enough to kill even with the rooms 100 % flooded, connected to the breach and no moving water

![image](https://user-images.githubusercontent.com/6137403/93246447-156aed80-f78d-11ea-972a-93ece2e4b5f8.png)

**Comment by Rokvach** [comment]
I was still able to cause quite a extensive flooding on the outpost as there are rooms the NPC's hardly ever goes in to, but I wasn't able to kill anyone with the flooding. On the pic below I highlighted the flooded rooms with blue and the breaches with red. Also attached a pic of my reputation to the image to show that I wasn't caught at all whilst plasma cutting the walls. 

I tried another go at flooding the outpost starting from the crew quarters top right but it was a lot harder as that area was a lot more heavily populated with NPC's that once I got a leak on the wall someone reported it and it was fixed. 

![floodedoutpost.jpg](https://images.zenhubusercontent.com/5c924117534a9b0a86cdddce/6319e07c-5392-48a9-a9f4-c95772e23d7a)

So you are only able to flood the outpost without anyone seeing you do it but you aren't able to kill anyone with the flooding now, I think this is enough, if some of the rooms had pumps in them I wouldn't of even been able to cause as much flooding as I was. But as there is no pumps anywhere else than the docking module its very easy to flood rooms as the water doesn't go anywhere. 

But I think this is a vast improvement and therefore working correctly. Closing.
